src/modules/universal-discovery/components/create/choose.language.component.js

```diff
@@ -26,6 +26,17 @@ export default class ChooseLanguageComponent extends Component {
             selectedLanguageCode = props.allowedLanguages[0];
         }
 
+        if (selectedLanguageCode === null || !props.languages.mappings[selectedLanguageCode].enabled) {
+            const languages = Object.values(props.languages.mappings);
+            const filteredLanguages = languages.filter((language) => {
```

Comment:
You can use `find` to not have to go through the whole array, just stop on the first hit.